src/text_render.rs

```diff
@@ -385,3 +553,29 @@ fn create_oversized_buffer(
 fn zero_depth(_: usize) -> f32 {
     0f32
 }
+
+#[cfg(feature = "custom-glyphs")]
+#[derive(Debug, Clone, Copy, PartialEq)]
+/// The input data to render a custom glyph
+pub struct CustomGlyphInput {
```

Comment:
nit: `CustomGlyphRasterizationRequest` or something

Reply:
How about just `RasterizationRequest`?

Reply:
Sure, I'd just like to distinguish between rasterization requests for regular glyphs vs. custom glyphs somehow